internal/cmd/kafka/command_topic_produce.go

```diff
@@ -31,7 +32,9 @@ func (c *command) newProduceCommand() *cobra.Command {
 		Annotations:       map[string]string{pcmd.RunRequirement: pcmd.RequireCloudLogin},
 	}
 
+	cmd.Flags().String("key-schema", "", "The ID or filepath of the message key schema.")
 	cmd.Flags().String("schema", "", "The ID or filepath of the message value schema.")
+	pcmd.AddKeyFormatFlag(cmd)
 	pcmd.AddValueFormatFlag(cmd)
 	cmd.Flags().String("references", "", "The path to the references file.")
```

Comment:
the reference file is for value schema i suppose? may need to update the description

Reply:
Ah, didn't think of this. We might need a `--key-references`?

Reply:
may confirm with dg team. not sure is it's supported

Reply:
Key serialization should be the same exact process as value serialization, so if it's supported for one it should be supported for both!